### ISSUE TITLE
[Feature] Notify users working in the system when the system is unexpectedly not available

### ISSUE THREAD

**Issue by beparticular** [CMSgov/eAPD#3236]
**Needs to be refined w/design/dev/product**

As a user, I'd like to know when the eAPD system is unexpectedly unavailable while I'm working, so I'm not confusing by things that appear to be broken in the system or potentially think I've made a change when I haven't. 

Currently users are only alerted when they are working inside an APD. Consider the information in this ticket to determine where to add the time out modal. Currently it only shows in the APD. It would also need to show when someone is

**Description and related issues -**
## If a user is logged into the system for unexpected downtime, the following happens:
If they are working in an APD, they are shown the "not saved" modal immediately upon not having a connection to the system. After 15 minutes they will be time out (although - this will change to a longer period of time soon). 

## If they are NOT working in an APD, the following happens: 
- If a user is trying to enter an APD - currently the system just shows "loading APD"
- If a user is trying to create an APD - currently the system just shows "loading APD"
- In the state affiliation request area - currently, you can get to this page and select a state, but if you hit submit on this page it goes to a blank page, if you choose “back to login” it doesn’t do anything
- In the state admin panel - currently if you go to the admin panel or change tabs on the panel with no wifi, it just says “loading” in the table. If you try to edit a role you get a blank page. If you try to revoke a user - currently the system shows the modal but the revoke button on the modal doesn't appear to do anything. 
- If the user receives the session expiring modal  ( sign out signs you out, stay signed in shows a brief spinner and then freezes)
- The log in page tells you your pw is incorrect, but we’re solving that in #3223
- You can also continue to view the review view, export a PDF, and submit to an email address while the system is down. 

## Still need to test, but I need to submit this issue before I turn my wifi off to test one more thing... 
- This worked on the timeout modal, but not the not saved modal - need to test again 
- They are able to log out while the not saved modal is showing - https://app.zenhub.com/workspaces/cms-eapd-5a3823ad3659e94d8ee569be/issues/cmsgov/eapd/3179
- Cannot log out while there is no internet (tested from manage account page)
- - Does the timeout modal work when wifi is off? 

### This task is done when…
This could include:

- [ ] Determine if there are any additional areas that appear to not work or be broken when the system times out
- [ ] Determine if the desired behavior is to show the "not saved" modal inside the system on all of these pages as we do now
- [ ] identify 508 considerations and/or recommendations
- [ ] Product approves the experience

**Comment by beparticular** [comment]
@thetif will check to see if we get a different error when your connection is down vs our system is down. this will help determine if we use the same modal, or if we have a different option.

**Comment by thetif** [comment]
Yes we should be able to handle the errors differently when the server is down, we should get a connection refused error and when their internet is out we should get a internet disconnected error.

**Comment by jeromeleecms** [comment]
System downtime has not been an issue; deprioritizing this until it's needed to be revisited.
